css/style.css

```
@@ -16,4 +17,175 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+.container {
+    margin: 0 auto;
+}
+
+h2, h3 {
+    font-weight: 300;
+
+}
+/*Header*/
+
+ header {
+    display: flex;  
+}
+
+.logo {
+    width: 2em;
+    padding-left: 2em;
+}
+
+nav {
+    display: flex;
+    padding: 1em 0;
+}
+
+ul {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+}
+
+li {
+    padding: 0 1em;
+    cursor: pointer;
+}
+
+li:first-child {
+    font-weight: bold;
+}
+
+li:hover {
```

```suggestion
li:focus,
li:hover {
```
When adding hover, we usually add focus too so keyboard users who use `Tab` to jump to the link get the style too